desktop/cmp/input/CodeInput.ts

```diff
@@ -43,6 +43,14 @@ export interface CodeInputProps extends HoistProps, HoistInputProps, LayoutProps
     /** True to focus the control on render. */
```

Comment:
This is a `desktop` component. I did not see a mobile version, but let me know if I just overlooked something.